test_team_techtitans.py

```diff
@@ -1,28 +1,85 @@
 """Unit test file for team techtitans"""
 import unittest
+import sys
+from unittest.mock import patch
```

Comment:
Why was it necessary to import this?

Reply:
SYS was used for maxsize method later in the code. I can probably delete the 4th line. I forgot to delete it from my last attempt.

Reply:
That's right, I don't see the need for 
```
from unittest.mock import patch
```

Reply:
It's been removed

Reply:
Made the change for the right format. I keep getting a score of Assertion score 0.5 instead of 0.85,

Reply:
If you look at the opensource [code](https://github.com/microsoft/presidio/blob/cc31bb61982311d7e2ad7e3b5334da797cdb6f98/presidio-analyzer/presidio_analyzer/predefined_recognizers/us_itin_recognizer.py#L16) for Presidio , the initial scores for each regex match are shown.  0.5 would be correct for the stronger match without any context word enhancement. 

If you use the debugger to inspect the results object you'll see some details about the decision process and how the score is increased when a context word is present. 

<img width="752" alt="image" src="https://github.com/user-attachments/assets/e7f99478-4b4e-490a-91d7-6fff019bab65">

You can also print it, for example:
```python
            # Test the valid ITIN format
            result = analyze_text(f'My ITIN is {itin}', ['US_ITIN'])
            print(f"Result for 'My ITIN is {itin}': {result}")
            if result:
                print(result[0].analysis_explanation)
```

So a score of 0.5 for "My abc is" would be be correct since there are no context keywords the score does not get enhanced.

Reply:
Does it have to have a score of 0.85?